sw/device/tests/sim_dv/i2c_device_tx_rx_test.c

```diff
@@ -185,12 +186,17 @@ bool test_main(void) {
     expected_data[i] = rand_testutils_gen32_range(0, 0xff);
   };
 
-  while (tx_empty_irq_seen == false) {
+  uint8_t tx_fifo_lvl;
+  CHECK_DIF_OK(dif_i2c_get_fifo_levels(&i2c, NULL, NULL, &tx_fifo_lvl, NULL));
+  // Write expected data to tx fifo.
```

Comment:
This comment looks a little bit out of place. It's in the middle of code that's checking the tx_fifo is empty

Reply:
It was indeed out of place. It is removed now.